src/styles/blocks/container.scss

```diff
@@ -0,0 +1,7 @@
+.container {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
```

Comment:
Ensure that the CSS variable `--column-count` is defined somewhere in your styles. If it's not defined, the grid layout might not work as expected.